pages/[400].vue

```diff
@@ -4,7 +4,7 @@ useMeta({
   meta: [
     {
       name: 'theme-color',
-      content: colors.lightblue,
+      content: colors.rlLightblue,
```

Comment:
I changed this file in my pull request so we might have to redo this change

Reply:
merged in main